llvm/tools/repo-fragments/RepoFragments.cpp

```diff
@@ -31,69 +89,71 @@ cl::opt<bool>
     UseComma("comma", cl::init(false),
              cl::desc("Output fields are comma (rather than CR) separated"));
 
-} // anonymous namespace
-
-raw_ostream &operator<<(raw_ostream &OS, pstore::index::digest const &Digest) {
-  return OS << Digest.to_hex_string();
-}
-
-static std::string getRepoPath() {
-  if (RepoPath.getNumOccurrences() == 0) {
-    // TODO: remove this envrionment variable once the matching behavior is
-    // removed from the compiler.
-    if (auto File = getenv("REPOFILE")) {
-      return {File};
-    }
-  }
-  return RepoPath;
-}
+} // end anonymous namespace
```

Comment:
Move the definition of the global variables as far down the source file as possible. This reduces the amount of code that can “see” them.